### PR TITLE
Bugfix/copy changes

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

image

### CODE REVIEW

**Comment by johnnieskywalker** [comment]
not sure if wrap this text ort make form wider ? contacted design team 

image

**Comment by johnnieskywalker** [comment]
thanks @peculiarity  
image

File: features/manageVault/ManageVaultView.tsx [hunk 1]
```diff
@@ -44,24 +45,25 @@ function TextWithCheckmark({ children }: WithChildren) {
...
-function ManageVaultMultiplyTransition({ stage }: ManageVaultState) {
+function ManageVaultMultiplyTransition({ stage, vault }: ManageVaultState) {
+  const { t } = useTranslation()
   return stage === 'multiplyTransitionEditing' ? (
     <Grid mt={-3}>
       <Grid variant="text.paragraph3" sx={{ color: 'text.subtitle' }}>
-        <TextWithCheckmark>Get up to 3x exposure with your ETH </TextWithCheckmark>
         <TextWithCheckmark>
-          A new UI specifically designed for multiplying your Vault
+          {t('vault-form.subtext.checkmark1', { token: vault.token })}
```
Comment (peculiarity) [inline features/manageVault/ManageVaultView.tsx:54]: `token.toUpperCase()` ? I see that little bit higher .. you used it like that

File: features/openVaultOverview/SelectVaultTypeModal.tsx [hunk 2]
```diff
@@ -50,9 +61,9 @@ export function SelectVaultTypeModal({ ilk, token, close, balance }: ModalProps<
           </Text>
           <Text variant="paragraph3" sx={{ color: 'text.muted', mb: '24px' }}>
             {t('select-vault-type.borrow.subtext', {
-              token,
-              balance,
               maxBorrow: balance.times(1000),
+              maxBorrowAmount,
+              token: token,
```
Comment (peculiarity) [inline features/openVaultOverview/SelectVaultTypeModal.tsx:66]: `token` only is ok

File: features/manageVault/ManageVaultView.tsx (hunk 1 above)
Comment (johnnieskywalker) [reply features/manageVault/ManageVaultView.tsx:54]: you're right looks good now but we cant be sure token is uppercase 
image

**Review by peculiarity — APPROVED** [review]

**Review by peculiarity — CHANGES_REQUESTED** [review]
Please apply all copy to other `common.json` files for the other languages

**Comment by paszkowskiDamian** [comment]
image

can we round it?

**Review by paszkowskiDamian — DISMISSED** [review]

**Comment by paszkowskiDamian** [comment]
Translations look good 👍

File: features/openVaultOverview/SelectVaultTypeModal.tsx [hunk 3]
```diff
@@ -9,11 +10,22 @@ import { Box, Divider, Grid, Text } from 'theme-ui'
...
+  token,
+  liquidationRatio,
+  close,
+  balance,
+}: ModalProps<Props>) {
   const { t } = useTranslation()
 
+  const exposureMultiplier = 1 + 1 / (liquidationRatio - 1.0)
+
+  const maxBorrowAmount = ((1 / liquidationRatio) * 100000).toFixed(2)
```
Comment (paszkowskiDamian) [inline features/openVaultOverview/SelectVaultTypeModal.tsx:27]: I think you should take `liquidationRatio` and `user token balance` from context.

image

`maxBorrowAmount` should be based on user balance

Comment (paszkowskiDamian) [reply features/openVaultOverview/SelectVaultTypeModal.tsx:27]: But I wondering whether it is possible to `useAppContext` in `useRedirectToOpenVault` 🤔

Comment (johnnieskywalker) [reply features/openVaultOverview/SelectVaultTypeModal.tsx:27]: This took me lot of time on Monday and after few trials I decided to stop overcomplicate it and did this way. Well both solutions have some pros and cons

**Review by peculiarity — APPROVED** [review]

**Pull request merged**